### ISSUE TITLE
to_sql doesn't seem to allow upload of df with some columns when the target table has a calculated column

### ISSUE THREAD

**Issue by qzoxec** [pandas-dev/pandas#60609]
I am trying to append a dataframe with several columns to a MS SQL table with a calculated field. I get the following error on attempt: 
"sqlalchemy.exc.ProgrammingError: (pyodbc.ProgrammingError) ('42000', '[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]The column "Fieldname" cannot be modified because it is either a computed column or is the result of a UNION operator. (271) (SQLExecDirectW); [42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Statement(s) could not be prepared. (8180)')"

This issue seems to have been addressed in the AWS sdk using the use_column_names parameter. See here - https://github.com/aws/aws-sdk-pandas/issues/2170 

That parameter doesn't seem to exist when using a sqlalchemy pyodbc connection. - "TypeError: to_sql() got an unexpected keyword argument 'use_column_names'"

Is there something similar that exists for a sqlalchemy pyodbc connection?

**Comment by Liam3851** [comment]
In SQL Server, just not including the computed column in the dataframe you are passing through to_sql should work, assuming you're using if_exists='append' (I assume you must be because you have a computed column, which pandas would not create on its own).

**Comment by qzoxec** [comment]
@Liam3851 You are correct, I had an extra field in the df I was trying to pasd. It works fine, thanks.
